src/um/audioclient.rs

```diff
@@ -51,21 +51,68 @@ pub const AUDCLNT_E_RAW_MODE_UNSUPPORTED: HRESULT = AUDCLNT_ERR!(0x027);
 pub const AUDCLNT_S_BUFFER_EMPTY: SCODE = AUDCLNT_SUCCESS!(0x001);
 pub const AUDCLNT_S_THREAD_ALREADY_REGISTERED: SCODE = AUDCLNT_SUCCESS!(0x002);
 pub const AUDCLNT_S_POSITION_STALLED: SCODE = AUDCLNT_SUCCESS!(0x003);
+pub const AUDCLNT_S_NO_SINGLE_PROCESS: SCODE = AUDCLNT_SUCCESS!(0x00d);
 ENUM!{enum AUDCLNT_BUFFERFLAGS {
     AUDCLNT_BUFFERFLAGS_DATA_DISCONTINUITY = 0x1,
     AUDCLNT_BUFFERFLAGS_SILENT = 0x2,
     AUDCLNT_BUFFERFLAGS_TIMESTAMP_ERROR = 0x4,
 }}
+ENUM!{enum AUDCLNT_STREAMOPTIONS {
+    AUDCLNT_STREAMOPTIONS_NONE = 0,
+    AUDCLNT_STREAMOPTIONS_RAW = 0x1,
+    AUDCLNT_STREAMOPTIONS_MATCH_FORMAT = 0x2,
+    AUDCLNT_STREAMOPTIONS_AMBISONICS = 0x4,
+}}
+STRUCT!{struct AudioClientProperties {
+    cbSize: UINT32,
+    bIsOffload: BOOL,
+    eCategory: AUDIO_STREAM_CATEGORY,
+    Options: AUDCLNT_STREAMOPTIONS,
```

Comment:
This is only supported win8.1 and later, not sure tho how to express this

Reply:
Yeah i'm not sure either. Do you know of any other areas on winapi where functionality is only supported in Win8 and above?

Reply:
Currently winapi doesn't do anything special for struct fields only available on newer Windows. It is something I'd like to make better in the future, but since existing structs just include the newer fields unconditionally, this is fine as is.

Reply:
Ok great, thanks for clarifying @retep998. Anything holding back this PR from merging now?